os/Kconfig

```diff
@@ -268,7 +268,15 @@ config PASS1_OBJECT
 	default ""
 
 endif # Build 2-pass
-endmenu # Build Setup
+
+config VERSION_BUILD_DATE
```

Comment:
This looks much similar with CONFIG_VERSION_BUILD_TIME.
This is for version but its format is date. So, let's just say BINARY_VERSION.
We can't guarantee the version will be the same as build date.